contrib/depends/funcs.mk

```diff
@@ -32,7 +32,7 @@ endef
 define fetch_file
     ( test -f $$($(1)_source_dir)/$(4) || \
     ( $(call fetch_file_inner,$(1),$(2),$(3),$(4),$(5)) || \
-      $(call fetch_file_inner,$(1),$(FALLBACK_DOWNLOAD_PATH),$(3),$(4),$(5))))
+      $(call fetch_file_inner,$(1),$(FALLBACK_DOWNLOAD_PATH),$(4),$(4),$(5))))
```

Comment:
Could you explain why this change and the changes in the packages were needed?

Reply:
Some download file names only contain the version, e.g. `contrib/depends/packages/eudev.mk` download file name is `v3.2.6.tar.gz`, which can clash with other packages and also isn't self describing, so it gets renamed to `eudev-v3.2.6.tar.gz` after download.

The fallback mirror has `eudev-v3.2.6.tar.gz` uploaded so the call for `fetch_file_inner` has to be updated in a way that the download file name is never used.